src/relay/src/createRequestHandler.js

```diff
@@ -44,33 +37,12 @@ export default function createRequestHandler(
     uploadables: ?Uploadables,
   ) {
     return Observable.create((sink: Sink) => {
-      const queryID = requestNode.text;
```

Comment:
We probably don't need `Observable` anymore?

Reply:
Why not? It's still handly for proper error propagation or things like `poll`.

Reply:
Why is Observable better in that case? I would expect it to work the same now, just removing the observable, and keeping `customFetcher(requestNode, variables, uploadables).then().catch()`

Reply:
Sort of explained in person - better errors propagation (`sink.error`) and ability to use `poll` which wouldn't work otherwise.